requirements/base.txt

```diff
@@ -21,6 +21,7 @@ newspaper3k==0.2.8
 
 git+git://github.com/DemocracyClub/dc_base_theme.git@0.3.11
 git+https://github.com/DemocracyClub/dc_signup_form.git@2.1.0
+git+https://github.com/DemocracyClub/design-system.git@python-package
```

Comment:
I've merged the PR now, so you can change this line to `git+https://github.com/DemocracyClub/design-system.git`.

This won't pin to a version though - I think that's fine while we're developing both systems side by side, but we'll want to pin it like this before merging:

`git+https://github.com/DemocracyClub/design-system.git@[tag-name]`